linode/manager
# refactor: [M3-8945] - Update `yup` from `0.32.9`  to `1.4.0`

## Description 📝

- Updates our validation library `yup` to the latest available version 📦 
- This bring improved type-safety, faster type checking, and better `.when(...)` syntax ✨ 
- This should unblock us from updating `@hookform/resolvers`
- I mostly just followed [this upgrade guide ](https://github.com/jquense/yup/issues/1906)

## Preview 📷

> [!note]
> There should not be any UI changes

## How to test 🧪

- Verify all automated tests pass ✅ 
- Check schemas for faulty/incorrect changes 👀 

## As an Author, I have considered 🤔

- 👀 Doing a self review
- ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- 🤏 Splitting feature into small PRs
- ➕ Adding a [changeset](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md#writing-a-changeset)
- 🧪 Providing/improving test coverage
- 🔐 Removing all sensitive information from the code and PR description
- 🚩 Using a feature flag to protect the release
- 👣 Providing comprehensive reproduction steps
- 📑 Providing or updating our documentation
- 🕛 Scheduling a pair reviewing session
- 📱 Providing mobile support
- ♿  Providing accessibility support

<br/>


## As an Author, before moving this PR from Draft to Open, I confirmed ✅
